docs/qa/v037/CometBFT.md

```diff
@@ -0,0 +1,172 @@
+
+The spike to more than 175 blocks/minute is due to a slow node catching up.
+
+The collective spike on the right of the graph marks the end of the load injection, when blocks become smaller (empty) and impose less on the network.
+This behavior is reflected in the following graph, which shows the number of transactions processed per minute.
+
+![total-txs](./img/200nodes_cmt037/total_txs_rate.png)
+
+The baseline experienced a similar behavior, shown in the following graphs, where the gradient of the curves show the rate of block and transactions per minute.
+
+Over a period of 2 minutes, the height goes from 477 to 524.
+This results in an average of 23.5 blocks produced per minute, a rate similar to this experiment.
+
+![heights-baseline](./img/200nodes_tm037/v037_r200c2_heights.png)
+
+Over a period of 2 minutes, the total goes from 64525 to 100125 transactions,
+resulting in 17800 transactions per minute. However, we can see in the plot that
+all transactions in the load are process long before the two minutes.
+If we adjust the time window when transactions are processed (approx. 90 seconds),
+we obtain 23733 transactions per minute, again similar to this experiment.
+
+![total-txs-baseline](./img/200nodes_tm037/v037_r200c2_total-txs.png)
+
+### Memory Resident Set Size
+
+The Resident Set Size of all monitored processes is plotted below, with maximum memory usage of 2GB.
+
+![rss](./img/200nodes_cmt037/memory.png)
+
+A similar behavior was shown in the baseline, presented next.
+
+![rss](./img/200nodes_tm037/v037_r200c2_rss.png)
+
+The memory of all processes went down as the load as removed, showing no signs of unconstrained growth.
+
+
+#### CPU utilization
+
+The best metric from Prometheus to gauge CPU utilization in a Unix machine is `load1`,
+as it usually appears in the
+[output of `top`](https://www.digitalocean.com/community/tutorials/load-average-in-linux).
+
+It is contained below 5 on most nodes, as seen in the following graph.
+
+![load1](./img/200nodes_cmt037/cpu.png)
+
+A similar behavior was seen in the baseline. 
+
+![load1-baseline](./img/200nodes_tm037/v037_r200c2_load1.png)
+
+
+## Test Results
+
+The comparison against the baseline results show that both scenarios had similar numbers and are therefore equivalent.
+
+A conclusion of these tests is shown in the following table, along with the commit versions used in the experiments.
+
+| Scenario | Date | Version | Result |
+|--|--|--|--|
+|CometBFT | 2023-02-14 | v0.37.0-alpha3 (bef9a830e7ea7da30fa48f2cc236b1f465cc5833) | Pass
+
+
+
+
+
+
+[\#9533]: https://github.com/tendermint/tendermint/pull/9533
+[\#9534]: https://github.com/tendermint/tendermint/pull/9534
+[\#9539]: https://github.com/tendermint/tendermint/issues/9539
+[\#9548]: https://github.com/tendermint/tendermint/issues/9548
+[\#9537]: https://github.com/tendermint/tendermint/issues/9537
+[\#9581]: https://github.com/tendermint/tendermint/issues/9581
```

Comment:
Are all of these links referenced in the preceding Markdown text?

Reply:
not really. removed.